compiler/src/iree/compiler/Codegen/LLVMCPU/test/check_ir_before_llvm_conversion.mlir

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
I think we should have two RUNs, one with the flag enabled, one disabled.

Reply:
Two options expect different results on the tests with dynamic shapes, so we'll need to split the tests into three files to do that (one with only static, one with dynamic and flag enabled, one with dynamic and flag disabled).

IMO `check_ir_before_llvm_conversion.mlir` and `check_ir_before_llvm_conversion_not_fail_unbound.mlir` should provide enough coverage for static and dynamic with flag enabled and disabled.